presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PruneUnreferencedOutputs.java

```diff
@@ -519,7 +519,7 @@ public PlanNode visitDistinctLimit(DistinctLimitNode node, RewriteContext<Set<Sy
                 expectedInputs = ImmutableSet.copyOf(node.getOutputSymbols());
```

Comment:
node.getDistinctSymbols() instead of node.getOutputSymbols() should be used for expectedInputs